main.go

```diff
@@ -156,14 +154,15 @@ func (c RejectionCondition) matchesUserText(p *payload) bool {
 	return c.UserTextMatch == "" || regexp.MustCompile(c.UserTextMatch).MatchString(p.UserText)
 }
 
+// Returns a rejection reason and error code if the payload should be rejected by this condition, otherwise nil.
```

Comment:
For a bit more clarity:

```suggestion
// Returns a rejection reason and error code if the payload should be rejected by this condition; otherwise returns `nil` for both results.
```

likewise below